core/src/main/java/greencity/controller/HabitController.java

```diff
@@ -133,4 +137,30 @@ public ResponseEntity<PageableDto<HabitDto>> getAllByTagsAndLanguageCode(
     public ResponseEntity<List<String>> findAllHabitsTags(@ApiIgnore @ValidLanguage Locale locale) {
         return ResponseEntity.status(HttpStatus.OK).body(tagsService.findAllHabitsTags(locale.getLanguage()));
     }
+
+    /**
+     * Method for creating Custom Habit.
+     *
+     * @param userId  {@link Long} - user id.
+     * @param request {@link AddCustomHabitDtoRequest} - new custom habit dto.
+     * @return dto {@link AddCustomHabitDtoResponse}
+     *
+     * @author Lilia Mokhnatska.
+     */
+    @ApiOperation(value = "Add new custom habit.")
+    @ResponseStatus(value = HttpStatus.CREATED)
+    @ApiResponses(value = {
+        @ApiResponse(code = 201, message = HttpStatuses.CREATED, response = AddCustomHabitDtoResponse.class),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND),
+    })
+    @PostMapping("/custom/{userId}")
+    public ResponseEntity<AddCustomHabitDtoResponse> addCustomHabit(
+        @PathVariable Long userId,
```

Comment:
Can every user create a habit for some other user (when `loggedUserId != userId`)?

Reply:
I changed it. Now, we can't